<?xml version="1.0" encoding="UTF-8"?><testrun name="CheckOutImplTest.testNoItem" project="shopping-app" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testNoItem" classname="com.vg.shoppingapp.service.CheckOutImplTest" time="0.055">
    <error>java.lang.NullPointerException&#13;
	at com.vg.shoppingapp.service.CheckOutImpl.lambda$total$0(CheckOutImpl.java:47)&#13;
	at java.util.stream.Collectors.lambda$summingDouble$19(Collectors.java:518)&#13;
	at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)&#13;
	at java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1548)&#13;
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)&#13;
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)&#13;
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)&#13;
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)&#13;
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)&#13;
	at com.vg.shoppingapp.service.CheckOutImpl.total(CheckOutImpl.java:46)&#13;
	at com.vg.shoppingapp.service.CheckOutImplTest.testNoItem(CheckOutImplTest.java:40)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:73)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:46)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:180)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:41)&#13;
	at org.junit.runners.ParentRunner$1.evaluate(ParentRunner.java:173)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:220)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
  </testcase>
</testrun>
